pkg/controller/tls/certs.go

```diff
@@ -129,20 +129,37 @@ func ProvisionCerts(req router.Request, resp router.Response) error {
 		return err
 	}
 
+	provisionedCerts := map[string]interface{}{}
+
+	for _, ep := range appInstance.Status.Endpoints {
+		if err := prov(req, leUser, ep.Address, appInstance.Name, appInstanceIDSegment, appInstance.Namespace); err != nil {
+			return err
+		}
+		provisionedCerts[ep.Address] = nil
+	}
+
 	// FIXME: use error list instead of exiting on first error
```

Comment:
What to actually fix this?  Should be simple, just collect []error and then return utilerrors.NewAggregate(errs)

Reply:
k8s.io/apimachinery/pkg/util/errors.NewAggregate

Reply:
Done, thanks, just overlooked that since the last time :grin: